Return the exported_program from transform to allow internal usage

cccclai opened the pull request:
Summary: It's mainly to make it easier to use internally...

Differential Revision: D64082731

haowhsu-quic reviewed (approved):
Looks good to me, thank you.

cccclai committed:
Return the exported_program from transform to allow internal usage (#6210)

Summary:

It's mainly to make it easier to use internally...

Differential Revision: D64082731
